argnorm/data/manual_curation/deeparg_curation.tsv

```diff
@@ -1,13 +1,14 @@
 Original ID	ARO	Gene Name in CARD	Description
-CP000034.1.gene2198.p01|FEATURES|ompF|multidrug|ompF	3000265	porin OmpF	
-AAB08925|FEATURES|tetU|tetracycline|tetU	3004650	tet(U)	
-CP000647.1.gene2517.p01|FEATURES|ompF|multidrug|ompF	3000265	porin OmpF	
-gi:504720116:ref:WP_014907218.1:|FEATURES|ompF|multidrug|ompF	3000265	porin OmpF	
-AM180355.1.gene2260.p01|FEATURES|ermC|MLS|ermC	3000250	ErmC	
-gi:501976562:ref:WP_012681429.1:|FEATURES|cystathionine_beta-lyase_patB|unclassified|cystathionine_beta-lyase_patB	3000025	patB	
+CP000034.1.gene2198.p01|FEATURES|ompF|multidrug|ompF	3000265	porin OmpF

```

Comment:
Was this intended? removing the last column?

Reply:
Sorry, this was not intended. The empty columns would have been removed by Excel when I was checking the manual curation.